app/code/Magento/Theme/view/frontend/layout/default_head_blocks.xml

```diff
@@ -12,7 +12,7 @@
         <script src="requirejs/require.js"/>
     </head>
     <body>
```

Comment:
just think introduced/replaced new block can affect to another old code 3rd-party rely on block head.additional

Reply:
or i missed understood